lib/posix/pthread.c

```diff
@@ -459,6 +459,18 @@ int pthread_create(pthread_t *th, const pthread_attr_t *_attr, void *(*threadrou
 	return 0;
 }
 
+int pthread_getconcurrency(void)
+{
+	return 0;
+}
+
+int pthread_setconcurrency(int new_level)
+{
+	ARG_UNUSED(new_level);
```

Comment:
It is not compliant if for example someone call this with a negative value, i.e. 

```
ERRORS
       pthread_setconcurrency() can fail with the following error:

       EINVAL new_level is negative.
```

Reply:
Ok - I can fix that.